<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="ColorInput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>361</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Color Input</string>
    </property>
    <widget class="QLineEdit" name="colorInputEdit">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="formatComboBox">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="previewButton">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>110</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Preview</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>401</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Your Monitor Preview</string>
    </property>
    <widget class="QFrame" name="colorPreviewFrame">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>251</width>
       <height>80</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>771</width>
      <height>351</height>
     </rect>
    </property>
    <property name="title">
     <string>Predictions</string>
    </property>
    <widget class="QFrame" name="prediction1Frame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>10</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Pre1</string>
     </property>
    </widget>
    <widget class="QComboBox" name="monitorTypeComboBox">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
